runtime/jvmMain/src/kotlinx/benchmark/jvm/JvmBenchmarkRunner.kt

```diff
@@ -45,16 +45,17 @@ fun main(args: Array<String>) {
     val jvmArgs = runtimeMXBean.inputArguments
     when {
         jvmArgs.any { it.contains("libasyncProfiler") } -> jmhOptions.forks(0)
-        config.forks == null -> jmhOptions.forks(1) 
-        config.forks > 0 -> jmhOptions.forks(config.forks) 
+        config.forks == null -> jmhOptions.forks(1)
+        config.forks > 0 -> jmhOptions.forks(config.forks)
     }
 
+    val reportFormat = ResultFormatType.valueOf(config.reportFormat.toUpperCase())
```

Comment:
This gives not very useful error in case of invalid value.
Can we validate the specified report format in plugin, before trying to run benchmarks?

Reply:
will do

Reply:
fixed